recipes-domd/domd-image-weston/files/meta-xt-prod-extra/doc/local.conf.rcar-domd-image-weston

```diff
@@ -411,7 +412,6 @@ DISTRO_FEATURES_append = " virtualization"
 DISTRO_FEATURES_remove = " x11 gtk gobject-introspection-data nfc irda zeroconf 3g"
 
 # Remove multimedia options if XT_RCAR_PROPRIETARY_MULTIMEDIA_DIR is not defined
-MACHINE_FEATURES_remove = " ${@bb.utils.contains('XT_RCAR_PROPRIETARY_MULTIMEDIA_DIR','','','multimedia', d)}"
```

Comment:
If neither v4l2-renderer or XT_RCAR_PROPRIETARY_MULTIMEDIA_DIR are in use, we will have multimedia present in MACHINE_FEATURES, is it ok? Other changes LGTM

Reply:
Good point, thanks!I reworked this part, and addressed my own concern regarding not clear dependency between vsp-renderer and multimedia. So I made "backward" dependency: 'multimedia' machine feature is enabled only if it is required by use case. Like multimedia package is provided or vsp-renderer is enabled.

Reply:
Also I rewrote commit message to describe this change. And add appropriate comment to place where this machine feature is enabled previously.

Reply:
Wrong decision. Will be rewritten.

Reply:
For now this block is returned to state as mentioned in first comment - just remove line that removes 'multimedia'.